lalrpop/src/normalize/lower/mod.rs

```diff
@@ -318,6 +318,30 @@ impl<'s> LowerState<'s> {
                 let arg_patterns = patterns(names.iter().map(|&(index, name, _)| (index, name)),
                                             symbols.len());
 
+
+                let action = {
+                    match norm_util::check_funky_expression(&action) {
```

Comment:
maybe call it `check_between_braces`? I **am** fond of the word `funky`, but this seems a bit more self-explanatory.